tests/yamls/tools/maestro_knowlege_mcp.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: maestro/v1alpha1
+kind: MCPTool
+metadata:
+  name: maestrok
```

Comment:
nit: I believe we've been using `maestro-k` as the short name for `maestro-knowledge`